src/OceanSeaIceModels/InterfaceComputations/roughness_lengths.jl

```diff
@@ -112,34 +119,35 @@ function TemperatureDependentAirViscosity(FT = Oceananigans.defaults.FloatType;
                                             convert(FT, C₃))
 end
 
+@inline compute_air_kinematic_viscosity(ν::Number, ℂ, 𝒬) = ν
+
 """ Calculate the air viscosity based on the temperature θ in Celsius. """
-@inline function (ν::TemperatureDependentAirViscosity)(θ)
+@inline function compute_air_kinematic_viscosity(ν::TemperatureDependentAirViscosity, ℂ, 𝒬)
+    T₀ = AtmosphericThermodynamics.air_temperature(ℂ, 𝒬)
     FT = eltype(ν.C₀)
-    T  = convert(FT, θ - celsius_to_kelvin)
-    return ν.C₀ + ν.C₁ * T + ν.C₂ * T^2 + ν.C₃ * T^3
+    T′ = convert(FT, T₀ - celsius_to_kelvin)
```

Comment:
This conversion to me is unclear. I can't really understand which of the variables is C and which is K just by looking at the code.

Reply:
I agree!